docs/src/pages/components/css-baseline/css-baseline.md

```diff
@@ -83,9 +84,22 @@ const theme = createTheme({
 });
 ```
 
-This website uses `darkScrollbar` when dark mode is enabled.
 Be aware, however, that using this utility (and customizing `-webkit-scrollbar`) forces MacOS to always show the scrollbar.
 
+This is a deprecated functionality.
```

Comment:
https://deploy-preview-29454--material-ui.netlify.app/components/css-baseline/#scrollbars

The text is hard to notice that it is deprecated. I suggest to change to something like this.

```md
### Scrollbars

> This API is deprecated, consider using [color-scheme](#color-scheme) instead.

The colors of the scrollbars can be customized to improve the contrast (especially on Windows). Add this code to your theme (for dark mode).

### Color scheme

This API is introduced in @mui/material (v5.0.7) for switching between "light" and "dark" modes of native components such as scrollbar, using the `color-scheme` css property. To enable it, you can set `enableColorScheme=true` as follow:


```